playbooks/roles/mattermost/templates/config.json.j2

```diff
@@ -32,15 +32,20 @@
         "EnforceMultifactorAuthentication": false,
         "EnableUserAccessTokens": false,
         "AllowCorsFrom": "",
+        "AllowCookiesForSubdomains": false,
         "SessionLengthWebInDays": 30,
         "SessionLengthMobileInDays": 30,
         "SessionLengthSSOInDays": 30,
         "SessionCacheInMinutes": 10,
+        "SessionIdleTimeoutInMinutes": 0,
         "WebsocketSecurePort": 443,
         "WebsocketPort": 80,
         "WebserverMode": "gzip",
         "EnableCustomEmoji": false,
         "EnableEmojiPicker": true,
+        "EnableGifPicker": false,
+        "GfycatApiKey": "2_KtH_W5",
+        "GfycatApiSecret": "3wLVZPiswc3DnaiaFoLkDvB4X0IV6CpMkj4tf2inJRsBY6-FnkT08zGmppWFgeof",
```

Comment:
Is this supposed to be public?

Reply:
No, it decidedly isn't.  No idea what I need to do now…

Reply:
Well, it turns out this actually _is_ supposed to be public – these are [upstream's default values](https://github.com/mattermost/mattermost-server/blob/f2c180390599e66fee2f1a8c1a4ab52eea920c51/config/default.json#L48-L49).  I've got no idea what this is, but now I don't care anymore.